### DISCUSSION TITLE
Need help in getting the downloadable or stream abled url of any song.

### DISCUSSION THREAD

**Discussion by ankydv** [sigma67/ytmusicapi#546] [Q&A]
By using **get_song**, I could fetch metadata of the song including the signature stamp. But how to get url from this sign stamp? 
I've tried decrypting it. But it doesn't work when I host the api in a cloud service like render. It seems like youtube detects the ip of clould service and sends a different signature stamp.

**Comment by sigma67** [comment]
please use the search function :)

https://github.com/sigma67/ytmusicapi/issues?q=is%3Aissue+stream+url
